validation/implicit_free_surface/geostrophic_adjustment_test.jl

```diff
@@ -34,23 +40,29 @@ function geostrophic_adjustment_simulation(free_surface)
     set!(model, v=vᵍ, η=ηⁱ)
     gravity_wave_speed = sqrt(g * grid.Lz) # hydrostatic (shallow water) gravity wave speed
     wave_propagation_time_scale = model.grid.Δxᶜᵃᵃ / gravity_wave_speed
-    simulation = Simulation(model, Δt=2wave_propagation_time_scale, stop_iteration=2)
+    simulation = Simulation(model, Δt=2wave_propagation_time_scale, stop_iteration=10000)
 
     return simulation
 end
 
-function run_and_analyze(simulation)
+function run_and_analyze(simulation, sim)
```

Comment:
?

Reply:
What is `sim` and how is it different from `simulation`?

Reply:
to save different jld2 files for different methods, changed to sim_number